UnitTests/CommonTestUtils/SubmoduleTestHelpers.cs

```diff
@@ -33,16 +34,14 @@ public static void UpdateSubmoduleStatusAndWaitForResult(ISubmoduleStatusProvide
             List<DetailedSubmoduleInfo> result = new List<DetailedSubmoduleInfo>();
             provider.StatusUpdated += Provider_StatusUpdated;
 
-            provider.UpdateSubmodulesStatus(
+            provider.UpdateSubmodulesStatusAsync(
                 workingDirectory: module.WorkingDir,
                 gitStatus: gitStatus);
 
             AsyncTestHelper.WaitForPendingOperations();
 
             provider.StatusUpdated -= Provider_StatusUpdated;
 
-            return;
```

Comment:
Keep, not required but separate the internal methods

Reply:
@gerhardol 
Are you requesting that I put the `return;` line back in so that it visually separates the implementation from the local method (`void Provider_StatusUpdated(object sender, SubmoduleStatusEventArgs e)`)?

Reply:
Yes
This should be documented

Reply:
Yes
This should be documented